Mathlib/Algebra/CharP/Subring.lean

```diff
@@ -33,4 +32,18 @@ instance subring (R : Type u) [Ring R] (p : ℕ) [CharP R p] (S : Subring R) : C
 instance subring' (R : Type u) [CommRing R] (p : ℕ) [CharP R p] (S : Subring R) : CharP S p :=
   CharP.subring R p S
 
+/-- The characteristic of a division ring is equal to the characteristic
+  of its center-/
+theorem center_CharP_iff {R : Type u} [DivisionRing R] {p : ℕ} :
```

Comment:
This should be called `charP_center_iff` and the direction of the iff should be flipped. Same for the one below.

Reply:
I think we can drop the double `charP` and also below